android/app/src/test/java/com/microsoft/codepush/common/UpdateManagerCommonTest.java

```diff
@@ -63,7 +62,7 @@ public void downloadFailsIfDeleteNewUpdateFolderPathFails() throws Exception {
         mockStatic(FileUtils.class);
         PowerMockito.doThrow(new IOException()).when(FileUtils.class, "deleteDirectoryAtPath", anyString());
         PowerMockito.doReturn(true).when(FileUtils.class, "fileAtPathExists", anyString());
-        codePushUpdateManager.downloadPackage(mock(JSONObject.class), mock(DownloadProgressCallback.class), mock(PackageDownloader.class));
+        codePushUpdateManager.downloadPackage("", mock(PackageDownloader.class));
```

Comment:
Could you please clarify why we have changed `mock(JSONObject.class), mock(DownloadProgressCallback.class)` to an empty string?

Reply:
@sergey-akhalkov, signature of the method has changed to take a String with package hash.